src/rt/profilegc.d

```diff
@@ -48,6 +48,9 @@ extern (C) void profilegc_setlogfilename(string name)
 public void accumulate(string file, uint line, string funcname, string type,
     size_t sz) @nogc
 {
+    if (sz == 0)
+        return;
+
```

Comment:
This is just an optimization, right? Also some more verbose commit message could help.